lib/liquid/ruby_tokenizer.rb

```diff
@@ -0,0 +1,28 @@
+module Liquid
+  class RubyTokenizer
```

Comment:
I've updated this to just be `Tokenizer`, following along the lines of liquid-c's implementation of it